site2/website-next/release-notes/versioned/pulsar-2.9.3.md

```diff
@@ -0,0 +1,213 @@
+- [fix][broker] Fix producerFuture not completed in ServerCnx#handleProducer [#14467](https://github.com/apache/pulsar/pull/14467)
+- [fix][broker] Fix race condition between timeout and completion  in `OpAddEntry` [#15233](https://github.com/apache/pulsar/pull/15233)
+- [fix][broker] Fix race condition in updating lastMarkDeleteEntry field [#15031](https://github.com/apache/pulsar/pull/15031)
+- [fix][broker] Fix rewind failed when ``redeliverUnacknowledgedMessages`` [#15046](https://github.com/apache/pulsar/pull/15046)
+- [fix][broker] Fix topic policy reader close bug [#14897](https://github.com/apache/pulsar/pull/14897)
+- [fix][broker] Fix typo in enum name and handle closing of the channel properly since writeAndFlush is asynchronous [#15384](https://github.com/apache/pulsar/pull/15384)
+- [fix][broker] Fix when nextValidLedger is null caused NPE [#13975](https://github.com/apache/pulsar/pull/13975)
+- [fix][broker] Fix wrong prompt exception when get non-persistent topic list without GET_BUDNLE permission [#14638](https://github.com/apache/pulsar/pull/14638)
+- [fix][broker] Fix wrong state for non-durable cursor [#14869](https://github.com/apache/pulsar/pull/14869)
+- [fix][broker] Fix wrong unit of NIC speed on linux [#15770](https://github.com/apache/pulsar/pull/15770)
+- [fix][broker] Fix 404 error msg not being returned correctly using http lookup [#14677](https://github.com/apache/pulsar/pull/14677)
+- [fix][broker] Follow up on race condition fixes in ManagedCursorImpl [#15067](https://github.com/apache/pulsar/pull/15067)
+- [fix][broker] Remove the loadbalance/bundle-data node [#13164](https://github.com/apache/pulsar/pull/13164)
+- [fix][broker] Return if reset in progress [#14978](https://github.com/apache/pulsar/pull/14978)
+- [fix][broker] Use dispatchRateLimiterLock to update dispatchRateLimiter [#15601](https://github.com/apache/pulsar/pull/15601)
+- [fix][broker] When skipping updating mark delete position, execute callback with executor to prevent deadlock [#15971](https://github.com/apache/pulsar/pull/15971)
+- [fix][broker] Expose configurationMetadataStore and localMetadataStore [#15661](https://github.com/apache/pulsar/pull/15661)
+- [fix][broker] Filter the virtual NIC with relative path [#14829](https://github.com/apache/pulsar/pull/14829)
+- [fix][broker] Fix MetadataStoreException$NotFoundException while doing topic lookup [#15633](https://github.com/apache/pulsar/pull/15633)
+- [fix][broker] Fix calculation in getNumberOfEntriesInStorage [#15627](https://github.com/apache/pulsar/pull/15627)
+- [fix][broker] Fix error log miss stack trace when create tenant fail [#14366](https://github.com/apache/pulsar/pull/14366)
+- [fix][broker] Fix resource group does not report usage [#15292](https://github.com/apache/pulsar/pull/15292)
+- [fix][broker] Fix the pid occupied check when use pulsar-daemon start or stop process [#15745](https://github.com/apache/pulsar/pull/15745)
```

Comment:
Is this item duplicate with #14701 in Line73?

Reply:
Oh, thanks! I will fix it ASAP.

Reply:
Fixed.